package.json

```diff
@@ -15,7 +15,13 @@
   "engines": {
     "node": ">=4"
   },
+  "size-limit": [
+    {
+      "path": "index.js"
+    }
+  ],
   "scripts": {
+    "size": "size-limit",
```

Comment:
It's useless having this as a separate script, because most likely nobody will run it manually. It should be part of the `test` script. Or better yet, run only on CI.

Reply:
👍 to running on CI as a separate script

Reply:
You'll run it separately to get details of which packages are causing the bloat (--why flag).

Reply:
Isn't it easier to just say run `npm run size -- --why` than to direct someone to do this from scratch when they need to? I wasted 30 minutes of time trying to figure out how to use this tool. By setting it up here we can prevent this sort of repetitive work.

What's the downside of adding the tool?

Reply:
if it's not enforced in the build, this tool is a matter of taste. I, for example, use a totally different tool for checking package sizes, `package-size`. I use it when I need to, but I don't intend to include it in every repo I use it in.

To clear it up about the downside, if it's not taking part in the process of developing this package (part of `test`, ci, etc), it's not part of a repository and just a handy helper for someone.

Reply:
> I wasted 30 minutes of time trying to figure out how to use this tool.

See the latest post below for installing and using it.

Reply:
Again, the point is when a new developer comes to this repo and is wondering how to do this. It's easier for them to run a npm/make command than figure out the instructions from scratch.

If you want to use package-size, you can still do that, or we could even update this PR to use package-size. This is exactly the point I'm trying to make - a new developer coming into this repo shouldn't have to figure out these things, we should make it easy for them to do this.

Reply:
I don't agree. It's not a complex task to execute, it doesn't require any explanation from us nor interaction with this repo specifically. `size-limit`'s docs are pretty good, their usage section (https://www.npmjs.com/package/size-limit#usage) clearly explains how to use it.

To sum up, we shouldn't include tools that are just for convenience for someone (not everyone) and if they are not affecting the project directly (by taking part in CI build or being part of `npm test`).